plugin-parser/mlops-plugin-parser.cc

```diff
@@ -238,6 +210,73 @@ _parse_json (const gchar *json_path, mlsvc_json_type_e json_type, const gchar *a
 /**
  * @brief Handle INSTALL process. Tizen app-installer invoke this function.
  */
+static void
+_get_json_config (const char *json_path, const gchar *app_info)
+{
+  g_autofree gchar *json_string = NULL;
+  g_autoptr (JsonParser) parser = NULL;
+  g_autoptr (GError) err = NULL;
+  g_autoptr (GList) members = NULL;
+  GList *iter;
+  JsonNode *root;
+  JsonObject *object;
+
+  if (!g_file_test (json_path, (GFileTest) (G_FILE_TEST_EXISTS | G_FILE_TEST_IS_REGULAR))){
+    _E ("The parameter, json_path, is invalid. It should be a valid string.");
+    return;
+  }
+
+  if (!g_file_get_contents (json_path, &json_string, NULL, NULL)){
+    _E ("Failed to read configuration file '%s'.", json_path);
+    return;
+  }
+
+  parser = json_parser_new ();
+  if (!parser){
+    _E ("Failed to parse configuration file, cannot allocate memory for JsonParser. Out of memory?");
+    return;
+  }
+    
+  if (!json_parser_load_from_data (parser, json_string, -1, &err)){
+    _E ("Failed to parse configuration file, cannot load json string (%s).",
+        err ? err->message : "Unknown error");
+    return;
+  }
+    
+  root = json_parser_get_root (parser);
+  if (!root){
+    _E ("Failed to parse configuration file, cannot get the top node from json string.");
+    return;
+  }
+
+  object = json_node_get_object (root);
+  members = json_object_get_members(object);
+
+  for (iter = members; iter !=NULL ; iter = iter->next) {
+    const gchar *name = (const gchar*)iter->data;
+    JsonNode *node;
+
+    if (g_ascii_strcasecmp (name, "models") == 0) {
```

Comment:
json node name:
"model" "pipeline" "resource" vs "models" "pipelines" "resources" 

or 

if (g_ascii_strcasecmp (name, "model") == 0 || g_ascii_strcasecmp (name, "models") == 0) {

Reply:
How about using "models", "pipelines", and "resources" as the default or recommend?
"models" json node has "model" value in sample json.

But, i modified the code that "model" "pipeline" "resource" are also possible!